conf/nd96amsr_v4.conf

```diff
@@ -24,37 +24,37 @@
  * || check_fs_mount_rw -t "devtmpfs" -s "devtmpfs" -f "/dev"
  * || check_fs_mount_rw -t "devpts" -s "devpts" -f "/dev/pts"
  * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run"
- * || check_fs_mount_rw -t "ext4" -s "/dev/root" -f "/"
  * || check_fs_mount_rw -t "securityfs" -s "securityfs" -f "/sys/kernel/security"
  * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/dev/shm"
- * || check_fs_mount_rw -t "tmpfs" -s "tmpfs" -f "/run/lock"
- * || check_fs_mount_ro -t "tmpfs" -s "tmpfs" -f "/sys/fs/cgroup"
  * || check_fs_mount_rw -t "pstore" -s "pstore" -f "/sys/fs/pstore"
  * || check_fs_mount_rw -t "efivarfs" -s "efivarfs" -f "/sys/firmware/efi/efivars"
+ * || check_fs_mount_rw -t "debugfs" -s "debugfs" -f "/sys/kernel/debug"
  * || check_fs_mount_rw -t "hugetlbfs" -s "hugetlbfs" -f "/dev/hugepages"
  * || check_fs_mount_rw -t "mqueue" -s "mqueue" -f "/dev/mqueue"
- * || check_fs_mount_rw -t "debugfs" -s "debugfs" -f "/sys/kernel/debug"
  * || check_fs_mount_rw -t "vfat" -s "/dev/sda15" -f "/boot/efi"
  * || check_fs_mount_rw -t "fusectl" -s "fusectl" -f "/sys/fs/fuse/connections"
  * || check_fs_mount_rw -t "configfs" -s "configfs" -f "/sys/kernel/config"
  * || check_fs_mount_rw -t "ext4" -s "/dev/sdb1" -f "/mnt"
  * || check_fs_mount_rw -t "tracefs" -s "tracefs" -f "/sys/kernel/debug/tracing"
+#  * || check_fs_mount_rw -t "nfs4" -s "*:/*" -f "/sched"
```

Comment:
Why are we adding these tests that are then commented out in both nd96 conf files?

Reply:
These are common with Cyclecloud. So I thought we should leave them there in the event a customer using cycle cloud would like to uncomment. Thoughts?